mobile_rewards/src/server.rs

```diff
@@ -405,12 +403,12 @@ async fn reward_period(client: &mut follower::Client<Channel>) -> Result<Range<u
     Ok(res.reward_height + 1..res.height)
 }
 
-pub fn get_time_range(last_reward_end_time: i64) -> (DateTime<Utc>, DateTime<Utc>) {
-    let after_utc = Utc.timestamp(last_reward_end_time, 0);
-    let now = Utc::now();
-    let stop_utc = now - Duration::minutes(DEFAULT_LOOKUP_DELAY);
-    let start_utc = after_utc.min(stop_utc);
-    (start_utc, stop_utc)
+pub fn get_time_range(
+    last_reward_end_time: i64,
+    now: DateTime<Utc>,
+) -> (DateTime<Utc>, DateTime<Utc>) {
+    let last_timestamp_utc = Utc.timestamp(last_reward_end_time, 0);
```

Comment:
So wouldn’t a second accuracy for file time stamp maybe loose the millisecond accuracy file time?

Reply:
This timestamp is not from a file, it is just a timestamp in seconds stored in the db